tavern/app.go

```diff
@@ -52,9 +53,36 @@ func newApp(ctx context.Context, options ...func(*Config)) (app *cli.App) {
 	app.Action = cli.ActionFunc(func(*cli.Context) error {
 		return run(ctx, options...)
 	})
+	app.Commands = []cli.Command{
+		{
+			Name: "redirector",
+			Usage: "Run a redirector connecting agents using a specific transport to the server",
+			Subcommands: []cli.Command{
+				{
+					Name: "http",
+					Usage: "Run an HTTP/1.1 redirector",
+					Action: func(cCtx *cli.Context) error {
+						// Convert main.Config options to redirector.Config options
+						redirectorOptions := []func(*redirector.Config){
+							func(cfg *redirector.Config) {
```

Comment:
can probably clean this up a bit but feel free to land and I'll make changes later on